ocaml/src/albamgr_client.ml

```diff
@@ -229,6 +229,23 @@ object(self)
         ~max:1 ~reverse:false >>= fun ((_, presets), _) ->
       Lwt.return (List.hd presets)
 
+    method list_presets2 ~first ~finc ~last ~reverse ~max =
+      client # query
+        ListPresets2
```

Comment:
This call should only be used in code paths related to this new functionality.
So only the new functionality shouldn't work.
However I now see that anywhere a preset gets refreshed will stop working...
I'll change the return type here so that instead of `'a` it returns `'a option` and deal with that at the call sites so we can get the desired behaviour.